### PR TITLE
Spotinst Functions - Add private/public support and Ruby template

### PR DESCRIPTION
**Pull request by guyklainer**
## What did you implement:

nodejs template - change default node version to 8.3
template - add support for private/public function
template - add ruby template

## How did you implement it:

changed the .yml template files and lib/plugins/create/create.js

## How can we verify it:

sls create -t spotinst-ruby

## Todos:

- [ ] Write tests
- [ ] Write documentation
- [ ] Fix linting errors
- [ ] Make sure code coverage hasn't dropped
- [ ] Provide verification config / commands / resources
- [ ] Enable "Allow edits from maintainers" for this PR
- [ ] Update the messages below

***Is this ready for review?:*** YES
***Is it a breaking change?:*** NO

### CODE REVIEW

File: tests/templates/test_all_templates [hunk 1]
```diff
@@ -25,3 +25,4 @@ integration-test google-nodejs
 integration-test spotinst-nodejs
 integration-test spotinst-python
 integration-test webtasks-nodejs
+integration-test spotinst-ruby
```
Comment (pmuens) [inline tests/templates/test_all_templates]: Could you add this under the other `spotinst` definitions?

File: docker-compose.yml [hunk 2]
```diff
@@ -79,6 +79,10 @@ services:
     image: python2.7
     volumes:
       - ./tmp/serverless-integration-test-spotinst-python:/app
+  spotinst-ruby:
+    image: python2.4.1
```
Comment (pmuens) [inline docker-compose.yml]: Why do we have a python image here? Does it include `Ruby` as well?

**Comment by guyklainer** [comment]
Hey, @pmuens,
We've made those changes and more in another PR from jeffnoehren.
It's under https://github.com/jeffnoehren/serverless

So actually you can close this PR. :)

Thanks!!

**Comment by pmuens** [comment]
Gotcha! Thanks for the update @guyklainer 👍

**Pull request closed without merging**